tests/forte/pipeline_with_record_check_test.py

```diff
@@ -0,0 +1,183 @@
+# Copyright 2021 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for Pipeline with Record Check functions-.
+"""
+
+import os
+import unittest
+from typing import Dict, List
+
+from forte.data.base_pack import PackType
+from forte.data.data_pack import DataPack
+from forte.evaluation.base import Evaluator
+from forte.common import ProcessExecutionException
+from forte.pipeline import Pipeline
+
+from tests.forte.pipeline_test import SentenceReader, DummyPackProcessor
```

Comment:
how about we put this test directly to the `pipeline_test`? Having some dependencies between tests may not be a good idea. We do have such imports before but I am considering removing them.